web/adrian01c/html/track/index.html

```diff
@@ -4,10 +4,18 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>track</title>
+    <title>Attributes in HTML</title>
   </head>
-  <body>
-    <h1>This is track</h1>
-    <h2>I want to create a dropdown menu but i have to use css and java for that</h2>
-  </body>
+  <body> 
+    <h1 style="background-color:DodgerBlue;">Did Romania melt the Russian tanks with a laser in 1968?</h1>
+    <h2 style="color: brown;">As the story goes, in November of 1968, Soviet forces were massing on Romania’s eastern border, in preparation for a military intervention. This was a reprisal prompted by Ceausescu’s defiant stance, particularly with regard to the Warsaw Pact invasion of Czechoslovakia earlier that year. That is when, supposedly, a top secret super powerful laser sprung into action, melting/vaporizing several tanks and aircraft, scaring the Soviets shitless and dissuading them from their belligerent intentions.</h2>
+    <img src="https://qph.fs.quoracdn.net/main-qimg-fccdedd9884c944d0b7f3b08ff3f0764.webp" alt="Real Laser Tank" width="600" height="200">
+    <button>See here more discussions on Quora about it</button> <link rel="stylesheet" 
```

Comment:
Should have a new line between `</button>` and `<link... >

Reply:
indeed